cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -367,6 +379,19 @@ private AuditLogEntry getLogEntry(ChannelHandlerContext ctx) {
     return logEntry;
   }
 
+  private String findResponseBody(ChannelBuffer channelBuffer, boolean isFirstChunk) {
+    if (!isFirstChunk) {
+      return channelBuffer.readable() ? channelBuffer.toString(Charsets.UTF_8) : "";
+    }
+    int bufferEnd = channelBuffer.writerIndex();
+    int start = channelBuffer.indexOf(channelBuffer.readerIndex(), bufferEnd, TWO_SEPERATOR_FINDER);
+
+    if (start >= 0) {
+      return channelBuffer.toString(start + 4, bufferEnd - start - 4, Charsets.UTF_8).trim();
```

Comment:
It is the length of the two line seperators '\r\n\r\n'. We do not want them to be included in the response body.